pkg/web/templates/components/add_apikey.html

```diff
@@ -0,0 +1,130 @@
+          </div>
+          <div class="my-2 grid grid-cols-2 items-center">
+            <label for="apikeys_view">apikeys:view</label>
+            <input type="checkbox" id="apikeys_view" name="permissions[]" value="apikeys:view" class="checkbox-style" />
+          </div>
+          <div class="my-2 grid grid-cols-2 items-center">
+            <label for="apikeys_revoke">apikeys:revoke</label>
+            <input type="checkbox" id="apikeys_revoke" name="permissions[]" value="apikeys:revoke" class="checkbox-style" />
+          </div>
+          <div class="my-2 grid grid-cols-2 items-center">
+            <label for="config_mange">config:manage</label>
+            <input type="checkbox" id="config_mange" name="permissions[]" value="config:manage" class="checkbox-style" />
+          </div>
+          <div class="my-2 grid grid-cols-2 items-center">
+            <label for="config_view">config:view</label>
+            <input type="checkbox" id="config_view" name="permissions[]" value="config:view" class="checkbox-style" />
+          </div>
+          <div class="my-2 grid grid-cols-2 items-center">
+            <label for="counterparties_manage">counterparties:manage</label>
+            <input type="checkbox" id="counterparties_manage" name="permissions[]" value="counterparties:manage" class="checkbox-style" />
+          </div>
+          <div class="my-2 grid grid-cols-2 items-center">
+            <label for="counterparties_view">counterparties:view</label>
+            <input type="checkbox" id="counterparties_view" name="permissions[]" value="counterparties:view" class="checkbox-style" />
+          </div>
+          <div class="my-2 grid grid-cols-2 items-center">
+            <label for="pki_manage">pki:manage</label>
+            <input type="checkbox" id="pki_manage" name="permissions[]" value="pki:manage" class="checkbox-style" />
+          </div>
+          <div class="my-2 grid grid-cols-2 items-center">
+            <label for="pki_view">pki:view</label>
+            <input type="checkbox" id="pki_view" name="permissions[]" value="pki:view" class="checkbox-style" />
+          </div>
+          <div class="my-2 grid grid-cols-2 items-center">
+            <label for="pki_delete">pki:delete</label>
+            <input type="checkbox" id="pki_delete" name="permissions[]" value="pki:delete" class="checkbox-style" />
+          </div>
+          <div class="my-2 grid grid-cols-2 items-center">
+            <label for="transactions_manage">transactions:manage</label>
+            <input type="checkbox" id="transactions_manage" name="permissions[]" value="transactions:manage" class="checkbox-style" />
+          </div>
+          <div class="my-2 grid grid-cols-2 items-center">
+            <label for="transactions_view">transactions:view</label>
+            <input type="checkbox" id="transactions_view" name="permissions[]" value="transactions:view" class="checkbox-style" />
+          </div>
```

Comment:
There are no transactions permissions, the travelrule permission manages transactions and secure envelopes. 

```suggestion
```

Reply:
Thanks! I think I originally typed `transactions` and instead of changing it to travel rule I ended up keeping both.